test/run-integration-tests.sh

```diff
@@ -1,21 +1,19 @@
 #!/bin/bash -e
 
-if ! test -d ../build ; then
-        echo "Expected build directory in ../build, but couldn't find it." >&2
-        exit 1
-fi
+BUILD_DIR="$($(dirname "$0")/../tools/find-build-dir.sh)"
 
-ninja -C ../build
+ninja -C "$BUILD_DIR"
 
 declare -A results
 
 RESULT=0
 FAILURES=0
 
+cd "$(dirname "$0")"
 for TEST in TEST-??-* ; do
         echo -e "\n--x-- Starting $TEST --x--"
         set +e
-        make -C "$TEST" BUILD_DIR=$(pwd)/../build clean setup run
+        make -C "$TEST" "BUILD_DIR=$BUILD_DIR" clean setup run
```

Comment:
i figure BUILD_DIR=$BUILD_DIR can be dropped entirely, no?

Reply:
Not really. BUILD_DIR is not exported in run-integration-tests.sh. We *could* export it, but I think it's nicer to pass it explicitly. Then it shown up in the logs, and it's also easier to c&p the command for testing.